demisto_sdk/tests/integration_tests/validate_integration_test.py

```diff
@@ -4,7 +4,7 @@
 from demisto_sdk.__main__ import main
 from demisto_sdk.commands.common import tools
 from demisto_sdk.commands.common.constants import DEFAULT_IMAGE_BASE64
-from demisto_sdk.commands.common.git_tools import git_path
+from demisto_sdk.commands.common.git_util import GitUtil
```

Comment:
I might have missed it, but should we add 1/2 cases for unified integrations and scripts validate flow?
Just as you've changes in these flows as well in `validate_manager` and I do not think they have integration tests.

Reply:
you might be right but I think this is unrelated to this change we should do this as a separate task in my opinion.

Reply:
As we had an issue with that last week, I think you should add it, but your call.